script/backport-patchset.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+import os
+import re
+import sys
+import urllib2
+
+SOURCE_ROOT = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
+PATCHES_DIR = os.path.join(SOURCE_ROOT, 'patches')
+
+def main():
+  if len(sys.argv) < 3:
+    print("Usage: backport-patchset.py [code-review-id] [patch-name]")
```

Comment:
Maybe [argparse](https://docs.python.org/2.7/library/argparse.html) could be useful here?